docs/index.md

```diff
@@ -167,9 +167,141 @@ Returning:
 </div>
 ```
 
-#### Slots (experimental)
+#### Slots
 
-_Slots are currently under development as a successor to Content Areas. The Slot APIs should be considered unfinished and subject to breaking changes in non-major releases of ViewComponent._
+_Slots now refer to V2 of the slot implementation, which is the successor to Slots V1 and Content Areas._
```

Comment:
Is it correct to assume that V2 is also subject to breaking changes? If so, it would be great if that could be noted here. 

Otherwise it's not clear to me when it would be a good time to adopt this API (or the whole gem for that matter). Personally, I'm still waiting for the slots API to settle so I can avoid unnecessary refactoring.

Reply:
> Is it correct to assume that V2 is also subject to breaking changes?

I don't believe so, unless some major issue with their design comes up.

> Otherwise it's not clear to me when it would be a good time to adopt this API (or the whole gem for that matter). Personally, I'm still waiting for the slots API to settle so I can avoid unnecessary refactoring.

I feel like view component has been pretty stable for a while now, with slots being marked as experimental.

Reply:
Thats encouraging and reasonable. 😄 

I would like to chime in that if V2 is not experimental any more and it is the natural successor of content areas, then this whole `Slots` section should move up above the `Content Areas` section. And the `Content Areas` section should be marked as deprecated.

Maybe I'm missing something, but that's how it looks to me :)

Reply:
This was a good call out, I just marked v2 as experimental as well.